source/reducer.js

```diff
@@ -57,6 +111,9 @@ export default (state = initialState, action = {}) => {
         authError: null,
         profile: null,
         data: {},
+        timestamp: {},
+        requesting: {},
```

Comment:
Why are these set to objects instead of:
```js
requesting: false,
requested: false,
```

Also same as above, are both parameters necessary?